check_copyright.py

```diff
@@ -599,9 +607,36 @@ def verify_config(config: configparser.ConfigParser) -> None:
         sys.exit(1)
 
 
+def git_diff_numstat() -> Dict[str, Tuple[int, int]]:
+    def call_git(args: List, die: bool = True) -> subprocess.CompletedProcess:
+        p = subprocess.run(['git'] + args, stdout=subprocess.PIPE,
+                           stderr=subprocess.STDOUT, check=die, text=True)
+        return p
+
+    numstat = {}
+
+    try:
+        call_git(['rev-parse', '--git-dir'])
+
+        p = call_git(['rev-parse', '--verify', 'HEAD'], die=False)
+        if not p.returncode:
+            against = 'HEAD'
+        else:
+            p = call_git(['hash-object', '-t', 'tree', os.devnull])
+            against = p.stdout.strip()
+
+        p = call_git(['diff', '--cached', '--numstat', against])
+        numstat = {f: (int(a), int(d)) for a, d, f in [l.split() for l in p.stdout.splitlines()]}
```

Comment:
It's a tiny thing, but I suggest longer names for better readability.

```suggestion
        numstat = {file: (int(added), int(deleted)) for added, deleted, file in [l.split() for l in p.stdout.splitlines()]}
```